cmd/replicate-add.go

```diff
@@ -64,6 +64,10 @@ var replicateAddFlags = []cli.Flag{
 		Name:  "replicate",
 		Usage: "comma separated list to enable replication of delete markers, and/or deletion of versioned objects.Valid options are \"delete-marker\", \"delete\" and \"\"",
 	},
+	cli.BoolFlag{
+		Name:  "disable-replica-sync",
```

Comment:
Are we set on this naming? It's a little unintuitive to have a flg `--disable-replica-sync=false` because of the double negative.

Would it be difficult to just do `replica-sync` and have it follow the 'enabled/disabled(default)' structure from minio-go? That would also keep the flag in line w/ replicate-edit.go

Reply:
This flag is set only when replica sync is explicitly disabled by user - if we were to have --replica-sync like in the edit command, there is the gray area of when it is not set. Since this applies to just 2 way replication, we can't have a edit to enforce this flag. Bool flags do not have a value - `replicate add --disable-replica-sync ` command disables this setting

Reply:
Ah I misunderstood from the example. If we do keep this, we might state something like:

"Usage: Disables syncing of replica metadata modifications in active-active replication configurations. Omit this flag to enable syncing replica metadata modifications.